tests/pages/LoginsOverviewPage.test.tsx

```diff
@@ -30,8 +33,52 @@ jest.mock("@src/dbClient", () => {
   };
 });
 
+(getFlashMessage as any).mockImplementation(() => {
+  return "Email address has been deleted";
+});
+
```

Comment:
Can I propose this one:

```suggestion
jest.mock("@src/utils/getFlashMessage", () => {
  return {
    getFlashMessage: jest.fn().mockImplementation(() => {
      return "Email address has been deleted";
    }),
  };
});
```

It gets rid of the `as any` and you don't need to import `getFlashMessage` and to mock it at the top of the file

Reply:
OK fixed 8bac2d7

Reply:
By the way, I decided to implement the `mockImplementation()` of `useCookies()` the same way in all concerned test files 1807fb9